runtime/doc/if_lua.txt

```diff
@@ -254,11 +254,58 @@ For example, to use the "nvim_get_current_line()" API function, call
 ------------------------------------------------------------------------------
 vim.* utility functions
```

Comment:
Not sure if this will be the final approach, but for now let's have two sections:

    vim.* builtin functions

    vim.* runtime functions

So that it is documented which functions require $VIMRUNTIME.

Reply:
So the functions from `src/nvim/lua/nvim.lua` are "builtin" and the ones from `runtime/...` are "runtime"? Just making sure :)